app/models/user_book.rb

```diff
@@ -2,5 +2,5 @@ class UserBook < ApplicationRecord
   belongs_to :book
   belongs_to :user
 
-  enum status: %i[available pending_requested pending_returned unavailable]
+  enum status: %i[available pending_requested unavailable]
 end
```

Comment:
Because this changed, we'll need to update the user_type to remove the pending_returned method and change the enum for unavailable_books to 2 instead of 3. I can do this on a new branch or if you want to just push it up to this pr that's fine too.